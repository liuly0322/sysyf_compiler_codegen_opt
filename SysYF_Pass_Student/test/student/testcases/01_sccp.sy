int main() {
    int count = 10;
    int i = 12;
    while (count > 0) {
        int x = i * 17;
        int j = i;
        i = j;
        count = count - 1;
        putint(i);
    }
    return 0;
}