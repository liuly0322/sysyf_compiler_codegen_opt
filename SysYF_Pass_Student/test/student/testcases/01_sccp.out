12121212121212121212
0